api/RTCPeerConnectionIceErrorEvent.json

```
@@ -185,6 +234,54 @@
               "version_added": "77"
             }
           },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": true
+          }
+        }
+      },
+      "port": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnectioniceerrorevent-port",
+          "support": {
+            "chrome": {
+              "version_added": "81"
+            },
+            "chrome_android": {
+              "version_added": "81"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "68"
+            },
+            "opera_android": {
+              "version_added": "58"
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": "13.0"
+            },
+            "webview_android": {
+              "version_added": "81"
+            }
+          },
           "status": {
             "experimental": false,
```

As following [the guideline](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status), this is too new and only has a single implementation, so it's `true`

```suggestion
            "experimental": true,
```

---

It's shipped in Safari too, so not experimental :)

---

I'm going to let it that way, then.